core-services/document-grounding/pom.xml

```diff
@@ -5,6 +5,7 @@
     <groupId>com.sap.ai.sdk</groupId>
     <artifactId>sdk-parent</artifactId>
     <version>1.6.0-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
```

Comment:
Drive-by change for all `AI Core services`
![image](https://github.com/user-attachments/assets/53c8dec2-2e33-464d-ac9a-2fac199d4df1)